site/src/components/banner.md

```diff
@@ -33,3 +33,23 @@ Ensure the user knows they need to wait for some event in order to interact with
 	<p>Loading banner.</p>
 </div>
 ```
+
+### Dismissing banner
+
+Banner can be dismissed in conjunction usage with `dismiss` component. Read more about it [here](./dismiss.md).
```

Comment:
I'd like to avoid links with generic text like "here." Perhaps something like this could work?

```suggestion
Banners can be made dismissable with [the `dismiss` component](./dismiss.md).
```

Reply:
Updated, thanks for recommendation :)